app/ui/src/scss/utils/_helpers.scss

```diff
@@ -79,3 +79,11 @@
 .syn-icon-small {
   @include syn-icon-box(24px);
 }
+
+.syn-scrollable {
+  @include syn-scrollable-parent;
+
+  .syn-scrollable--body {
+    @include syn-scrollable-body;
+  }
+}
```

Comment:
Using compound SASS selectors will save you some extra coing and will infor better of the intent of each selector:

```css
.syn-scrollable {
  @include syn-scrollable-parent;

  &--body {
    @include syn-scrollable-body;
  }
}
```

Also, bear in mind the level of CSS specificity generated here. Are you sure you want to constrain `.syn-scrollable--body` to have a parent `.syn-scrollable` classname wrapping it in order to execute it?

Ideally this is not how _modifiers_ operate in BEM. A CSS class with a modifier suffix should be able to run as a standalone class.

Reply:
Thanks, and you're right, decoupled those 2 elements.